### PR TITLE
Add AODtoDisk parameter to dataset config

### PR DESCRIPTION
**Pull request by germanfgv**
Adds ability to disable AOD disk copy using the parameter `aod_to_disk`

### CODE REVIEW

**Comment by germanfgv** [comment]
@jeyserma @jhonatanamado what do you think?

**Comment by jhonatanamado** [comment]
Hi @germanfgv , other than the previous comments, the changes looks good. Please do not forget to update the ProdConfig file.

**Comment by germanfgv** [comment]
I'll remove the changes to replay config from this PR. The changes to ProdConfig will be included in a separate PR, once we determine which datasets should be modified. This is only for the feature.

File: src/python/T0/RunConfig/Tier0Config.py [hunk 1]
```diff
@@ -215,6 +211,12 @@
             |
             |--> DiskNode - The disk PhEDEx node (should be T1 _Disk)
             |
+            |--> RAWTapeNode - The tape PhEDEx node for RAW (default None)
+            |
+            |--> RAWToDisk - Do we subscribe RAW to disk?
+            |
+            |--> AODToDisk - Do we subscribe RAW to disk?
```
Comment (jeyserma) [inline src/python/T0/RunConfig/Tier0Config.py:218]: RAW->AOD

**Comment by jeyserma** [comment]
No other comments from my side

**Pull request merged**